### DISCUSSION TITLE
Reminder: submission deadline is today

### DISCUSSION THREAD

**Discussion by gasse** [ds4dm/ml4co-competition#105] [Announcements]
Dear participants,

This is a reminder that today, October 28th, is the last day to send us your codes for the primal, dual and config tasks. We will accept submissions received until 23:59 AoE ([Anywhere on Earth](https://time.is/Anywhere_on_Earth)). We will evaluate only the latest submission received from each team. Only if there is a technical issue (with either installation or execution of your codes) will we allow for additional submissions past the deadline, on request from our side.

As a reminder also, for this last evaluation round we will process the entire test set, while in the previous rounds we were only evaluating 20% of it. Therefore the final results in the leaderboard will most likely change, even if you haven't updated your submission. This last evaluation will decide on the final ranking of the competition.

Good luck with your submissions !

Best,
The organizing team

**Comment by JeremyAlain** [comment]
Hi, 
a quick question. On the website, it says that the final results and winners will be announced on 21 November. Will however the leader board be updated before that either with the public test set (without the 20% hidden samples) or with the full test set? 
I would assume the answer is no, but I just thought I'd ask anyway. 

Cheers 
Jérémy

**Reply by gasse** [reply]
Hi @JeremyAlain ,

The next update of the leaderboard with be with the final results, with the full test set (100% of the hidden instances). November 21st is our tentative schedule to release the results (and thereby declare the winners), but it might be that we release the results before that date if we have them, or after if we encounter difficulties during the evaluation process. Anyway, we are now focused on the complete evaluation on the full test set, and we will not update the leaderboard with partial evaluations any more.

Best,
Maxime
